packages/shared/src/hooks/utils/useFeatureTheme.ts

```diff
@@ -0,0 +1,64 @@
+import { useLayoutEffect, useMemo } from 'react';
+import { useRouter } from 'next/router';
+import { ThemeMode, useSettingsContext } from '../../contexts/SettingsContext';
+import { useFeature } from '../../components/GrowthBookProvider';
+import { feature } from '../../lib/featureManagement';
+
+interface UseFeatureThemeResult {
+  version: number;
+  logo?: string;
+  logoText?: string;
+  body?: Record<string, string>;
+  navbar?: string;
+  cursor?: string;
+}
+
+export const useFeatureTheme = (): UseFeatureThemeResult | undefined => {
+  const { themeMode } = useSettingsContext();
+  const featureTheme = useFeature(feature.featureTheme);
+  const router = useRouter();
+  const isOnboarding = router?.pathname?.startsWith('/onboarding') ?? false;
+  const useTheme = !isOnboarding && !!featureTheme;
```

Comment:
Are u expecting for the featureTheme to be null/undefined as default with this evaluation?

Reply:
Growthbook does not allow that for JSON feature values, so you should adopt for a default value of `{}`

Reply:
Didn't know that, good catch. Will rewrite to make it work with the default growthbook forces on us.